docs/production/authentication-methods.md

```diff
@@ -518,6 +518,65 @@ to the root and `engineering` subdomains:
 </saml2:Attribute>
 ```
 
+### Using Keycloak as a SAML IdP
+
+1. Make sure you reviewed [this article][saml-help-center], which
+   details how to configure Keycloak properly to use SAML with Zulip.
+2. Verify that `SOCIAL_AUTH_SAML_ENABLED_IDPS['entity_id']` and
+   `SOCIAL_AUTH_SAML_ENABLED_IDPS['url']` are correct in your Zulip
```

Comment:
This isn't quite correct - the keys of `SOCIAL_AUTH_SAML_ENABLED_IDPS` are idp names, and then `entity_id` etc. are keys of the result idp_dict.
```
SOCIAL_AUTH_SAML_ENABLED_IDPS: Dict[str, Any] = {
    ## The fields are explained in detail here:
    ##     https://python-social-auth.readthedocs.io/en/latest/backends/saml.html
    "idp_name": {
        ## Configure entity_id and url according to information provided to you by your IdP:
        "entity_id": "https://idp.testshib.org/idp/shibboleth",
        "url": "https://idp.testshib.org/idp/profile/SAML2/Redirect/SSO",
```

Reply:
Yeah, agreed.  I have no more time today, feel free to fix and repush.